app/assets/stylesheets/views/comments.scss

```diff
@@ -275,3 +275,48 @@
 .moderator-template-button:not(.active) {
   @extend .crayons-btn--outlined;
 }
+
+#toggle-comments-sort-dropdown {
+  cursor: pointer;
+}
+
+#comments-sort-dropdown {
```

Comment:
Refactor (blocking): Styling for a dropdown is not required. Use the CSS class `crayons-dropdown`. Using this class wires up a bunch of functionality to the dropdown to make it more accessible as well.

For margins and padding stick to our CSS variables, e.g. ✅  `padding: var(--su-4);` ❌  `margin-top: 0.25rem;`.

@ludwiczakpawel, do you have any design input here?

Reply:
I know im late to the party, but it seems like it has been fixed...?

Reply:
yes, I was waiting on test cases to inform you guys